### ISSUE TITLE
Does not work with go modules because of : in filename

### ISSUE THREAD

**Issue by karel-3d** [JohannesKaufmann/html-to-markdown#9]
With new go+go modules, I get

```
go get github.com/JohannesKaufmann/html-to-markdown: no matching versions for query "upgrade"
```

:-(

**Comment by karel-3d** [comment]
Maybe related to this?

https://github.com/golang/go/issues/34539

**Comment by karel-3d** [comment]
It is because : in filename in the tests. Unzipping fails and the whole thing fails

**Comment by karel-3d** [comment]
If you get stuck on this like me, get this instead  (fork of this repo that is not dead)

thx @Skarlso  

https://github.com/Skarlso/html-to-markdown

**Comment by Skarlso** [comment]
Thanks. :)

**Comment by JohannesKaufmann** [comment]
fixed with #5
